feature-libs/checkout/base/ng-package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "../../../node_modules/ng-packagr/ng-package.schema.json",
+  "lib": {
+    "entryFile": "./public_api.ts",
+    "umdModuleIds": {}
```

Comment:
I didn't build it, but I'm assuming it would throw warnings for each dependencies that encapsulates the sub-lib entrypoint

Reply:
But if I'm wrong, I'm sure we can safely remove the property

Reply:
It doesn't print any warning 🤷🏻‍♂️
Removing.